lib/app/docker/imageservice.go

```diff
@@ -138,21 +138,26 @@ func newImageService(req RegistryConnectionRequest) (*imageService, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	// Strip the scheme prefix if specified in the address as we manipulate
+	// the registry address without the URL scheme when wrapping/unwrapping image references
+	registryPrefix := strings.TrimPrefix(req.RegistryAddress, "http://")
```

Comment:
The problem with `make test` is that it executes the tests from within a container which skips the tests tha require Docker themselves, hence I stumbled on a couple of failing tests.
But this seems to only affect 6.1.x - i.e. the unit tests are not failing for 7.0.x (and possibly for master) which makes me think I may need to look into that in more detail.